evalml/tests/automl_tests/test_automl.py

```diff
@@ -1496,9 +1513,10 @@ def test_describe_pipeline(mock_fit, mock_score, return_dict, caplog, X_y_binary
 
 @patch("evalml.pipelines.BinaryClassificationPipeline.score")
 @patch("evalml.pipelines.BinaryClassificationPipeline.fit")
+@patch("evalml.tuners.skopt_tuner.Optimizer.tell")
```

Comment:
Surprisingly, `Optimizer.tell` uses up a lot of the time for tests that run automl for many iterations.